easy-ui-react/src/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,66 @@
+@use "../styles/common" as *;
+@use "../InputField/mixins" as Input;
+@use "../Menu/mixins" as Menu;
+
+.DatePicker {
+  @include Input.root;
+}
+
+.datePickerTrigger {
+  display: flex;
+  align-items: center;
+  @include Input.input;
+
+  &:hover {
+    @include Input.hovered;
+  }
+  @include Input.iconEndInput;
+}
+
+.datePickerTriggerContainer {
+  @include Input.inputIconContainer;
+}
+
+.datePickerSm .datePickerTrigger {
+  @include Input.inputSm;
+  @include Input.iconEndInput;
+}
+
+.errorInput {
+  @include Input.error;
+}
+
+.underlay {
+  position: fixed;
+  inset: 0;
+}
+
+.dialog {
+  background: theme-token("color.neutral.000");
+  border-radius: design-token("shape.border_radius.lg");
+  box-shadow: design-token("shadow.modal");
+  pointer-events: auto;
+  outline: none;
+  position: relative;
+  z-index: design-token("z-index.input_icon");
```

Comment:
**suggestion**: up to you and a minor thing, but i think we could just use the value of 1 here or define a component level token. _generally_ so far we've been assigning z-index tokens in association with some semantics